### PR TITLE
Count of teams

### PR DESCRIPTION
**Pull request by taphill**
PR to merge count_of_teams into Iteration 2 branch
- Add csv dummy file for count_of_teams
- Create test for count_of_teams_method
- Create count_of_teams method

### CODE REVIEW

**Review by smcunning — CHANGES_REQUESTED** [review]
We might want to refactor this to use map

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -34,6 +34,15 @@ def lowest_total_score
   end
 
 #------------LeagueStatistics
+  def count_of_teams
+    teams = []
+    games.each do |game|
+      teams << game.away_team_id unless teams.include? game.away_team_id
+      teams << game.home_team_id unless teams.include? game.home_team_id
```
Comment (Stearnzy) [inline lib/stat_tracker.rb]: Could use the uniq method here instead to take out duplicates.  To use this, you'd have to have an array of all the teams, so I'm not sure if that would make things more computationally expensive.

Comment (taphill) [reply lib/stat_tracker.rb]: Updated solution to make use of uniq!

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -35,11 +35,11 @@ def lowest_total_score
 
 #------------LeagueStatistics
   def count_of_teams
-    teams = []
-    games.each do |game|
-      teams << game.away_team_id
-      teams << game.home_team_id
+    teams = games.flat_map do |game|
```
Comment (smcunning) [inline lib/stat_tracker.rb:38]: Way cool you found flat_map!

**Review by Stearnzy — COMMENTED** [review]
Looks great!!  Clean work!

**Review by smcunning — APPROVED** [review]

**Pull request merged**